.github/workflows/release.yml

```diff
@@ -124,6 +124,31 @@ jobs:
           name: enarx-x86_64-windows-msi
           path: target\wix\enarx-x86_64-windows.msi
 
+  sign-x86_64-enarx-keep:
+    needs: [ build-x86_64 ]
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: enarx-x86_64-unknown-linux-musl
+    - name: Generate the enarx keep signature
+      env:
+        SEV_ID_KEY: ${{ secrets.SEV_ID_KEY }}
+        SEV_ID_KEY_SIGNATURE_BLOB: ${{ secrets.SEV_ID_KEY_SIGNATURE_BLOB }}
+        SGX_KEY: ${{ secrets.SGX_KEY }}
+      run: |
+        echo -e "$SEV_ID_KEY" | base64 --decode | gunzip > sev-id.key
+        echo -e "$SEV_ID_KEY_SIGNATURE_BLOB" | base64 --decode | gunzip > sev-id-key-signature.blob
+        echo -e "$SGX_KEY" | base64 --decode | gunzip > sgx.key
```

Comment:
Why `echo -e`?? Did you mean `echo -n`??

Reply:
fixed.